packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { AssertionError, expect } from "@assertive-ts/core";
+import { render } from "@testing-library/react-native";
+import {
+  View,
+  TextInput,
+} from "react-native";
+
+import { ElementAssertion } from "../../src/lib/ElementAssertion";
+
+describe("[Unit] ElementAssertion.test.ts", () => {
+  describe(".toBeDisabled", () => {
+    context("when the element is TextInput", () => {
+      it("returns the assertion instance when is not editable", () => {
+        const element = render(
+            <TextInput testID="id" editable={false} />,
+        );
+        const test = new ElementAssertion(element.getByTestId("id"));
+        expect(test.toBeDisabled()).toBe(test);
+      });
+      it("throws an error when it is editable", () => {
+        const reactElement = render(<TextInput editable={true} testID="id" />);
+        const test = new ElementAssertion(reactElement.getByTestId("id"));
+
+        expect(() => test.toBeDisabled())
+            .toThrowError(AssertionError)
+            .toHaveMessage('Received element <TextInput testID="id"... /> is enabled.');
+      });
+    });
+
+    context("when the parent has property aria-disabled", () => {
+      it("returns disable for parent and child element when aria-disabled=true", () => {
+        const element = render(
+            <View aria-disabled={true} testID="parentId">
+              <View testID="childId">
+                <TextInput />
+              </View>
+            </View>,
+        );
```

Comment:
Use 2-space indentation:
```suggestion
        const element = render(
          <View aria-disabled={true} testID="parentId">
            <View testID="childId">
              <TextInput />
            </View>
          </View>,
        );
```

Reply:
Can you check the rest of the file for indentation spaces? Thanks!